lib/generators/debugs_bunny/install/install_generator.rb

```diff
@@ -7,7 +7,7 @@ class InstallGenerator < Rails::Generators::Base
     def install
       model_name = 'DebugTrace'
       generate "debugs_bunny:trace #{model_name}"
-      generate "debugs_bunny:migration:create_traces --table_name #{model_name}"
```

Comment:
I removed the functionality to define a custom table name; this is necessary so that the rake task knows a priori the name of the table it needs to clear